framework/src/modules/interoperability/mainchain/commands/sidechain_registration.ts

```diff
@@ -277,13 +276,10 @@ export class SidechainRegistrationCommand extends BaseInteroperabilityCommand {
 			});
 	}
 
-	protected getInteroperabilityInternalMethod(
-		context: StoreGetter | ImmutableStoreGetter,
-	): MainchainInteroperabilityInternalMethod {
+	protected getInteroperabilityInternalMethod(): MainchainInteroperabilityInternalMethod {
```

Comment:
I think it's better to just instantiate in the module and pass it similar to other method